ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go

```diff
@@ -1862,6 +1862,74 @@ spec:
 	assert.NotEqual(t, nmState, nil)
 }
 
+func Test_managedclusterLabel(t *testing.T) {
```

Comment:
Might also be worth having a test where no labels are included in the SiteConfig.

Reply:
Yep! Added a new test for that.